deploy/qserv.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Install and uninstall Qserv operator
+
+set -euxo pipefail
+
+usage() {
+    cat << EOD
+
+Usage: `basename $0` [options] [cmd]
+
+  Available options:
+    -h, --help           This message
+    -l, --local          Install from local git repository
+    --n NAMESPACE        Specify namespace (default: kube-system)
+    --no-kubedb          Do not install KubeDB operator
```

Comment:
are we sure we want this to be a negative option? or is the more common case `--install-kubedb`?

Reply:
Please be aware that kubedb is a Redis pre-requisite, as requested "not-install" is now the default.